### PR TITLE
Task 8 testing with client

### PR DESCRIPTION
**Pull request by VasiaBirchak**

### CODE REVIEW

**Comment by VasiaBirchak** [comment]
Is it ok if my test_logout_page: Tests that a 302 status code is returned after the logout is done because a redirect happens after a successful logout?

File: myblog/blog/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@pytest.mark.django_db
+class TestUserAuthViews:
+    @pytest.fixture()
+    def user(self, db):
+        return User.objects.create_user(username="testuser", password="1234")
+
+    def test_login_page(self, client):
+        url = reverse('user_login')
+        response = client.get(url)
+        assert response.status_code == 200
```
Comment (ejariza-evowill) [inline myblog/blog/tests/test_views.py:15]: Dont check only status code, check here for example that user was logged in.

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]

**Comment by VasiaBirchak** [comment]
Done the tests

File: myblog/blog/tests/test_views.py [hunk 2]
```diff
@@ -9,18 +9,33 @@ class TestUserAuthViews:
...
         url = reverse('user_reg')
         response = client.get(url)
         assert response.status_code == 200
+        response = client.post(url, {
+            'username': 'newuser',
+            'password1': 'complexpassword',
+            'password2': 'complexpassword',
+            'email': 'newuser@example.com'
+        })
+        assert response.status_code == 302
```
Comment (ejariza-evowill) [inline myblog/blog/tests/test_views.py:33]: for redirections you can use the `follow=True`, parameter in in client calls

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]

**Review by ejariza-evowill — APPROVED** [review]

**Pull request merged**